src/client/stylesheets/card.css

```diff
@@ -46,6 +46,9 @@ div.collapsible-header p{
 ul.collapsible{
   list-style-type: none;
   list-style-image: none;
+	height: 200px;
```

Comment:
give it a max-height instead so once it exceeds a certain height the scroll kicks in